### PR TITLE
Add documentation for SmartTub services

### PR DESCRIPTION
**Pull request by mdz**
## Proposed change

Two new services for updating the filtration cycle settings.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/46980
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/smarttub.markdown [hunk 1]
```diff
@@ -24,3 +24,28 @@ The `smarttub` integration allows you to view and control hot tubs which use the
...
+## Services
+
+### Service `smarttub.set_primary_filtration`
+
+This service allows you to update the settings for the primary filtration cycle
+on a hot tub.
+
+| Service data attribute | Optional | Description | Example |
+| ---------------------- | -------- | ----------- | ------- |
+| `entity_id` | no | The entity to update. | sensor.jacuzzi_j_335_primary_filtration_cycle
```
Comment (frenck) [inline source/_integrations/smarttub.markdown]: When using registering entity services, the entity becomes optional (as an area, device or all can be targeted instead).

```suggestion
| `entity_id` | yes | The entity to update. | sensor.jacuzzi_j_335_primary_filtration_cycle
```

File: source/_integrations/smarttub.markdown [hunk 2]
```diff
@@ -24,3 +24,28 @@ The `smarttub` integration allows you to view and control hot tubs which use the
...
+
+
+### Service `smarttub.set_secondary_filtration`
+
+This service allows you to update the settings for the secondary filtration
+cycle on a hot tub.
+
+| Service data attribute | Optional | Description | Example |
+| ---------------------- | -------- | ----------- | ------- |
+| `entity_id` | no | The entity to update. | sensor.jacuzzi_j_335_secondary_filtration_cycle
```
Comment (frenck) [inline source/_integrations/smarttub.markdown]:
```suggestion
| `entity_id` | yes | The entity to update. | sensor.jacuzzi_j_335_secondary_filtration_cycle
```

**Review by frenck — APPROVED** [review]
Thanks, @mdz 👍

**Comment by mdz** [comment]
Thanks! I'm curious, why should `entity_id` be documented as optional in this case?

**Comment by frenck** [comment]
Because they are entity services, which can be targeted using an area or device as well

**Comment by mdz** [comment]
Ah, thanks for explaining, will keep this in mind in the future

**Pull request merged**